core/services/commander/main.py

```diff
@@ -204,7 +206,33 @@ async def root() -> Any:
     return HTMLResponse(content=html_content, status_code=200)
 
 
+def setup_ssh() -> None:
+    # store the key in the docker .config volume
+    key_path = "/root/.config/.ssh/"
+    # authorized_keys is in the pi home directory, we just add our key to it
+    authorized_keys_path = "/home/pi/.ssh/authorized_keys"
+
+    try:
+        Path("key_path").mkdir(parents=True, exist_ok=True)
+        # check if id_rsa.pub exists, creates a new one if it doesnt
+        if not os.path.exists(Path(key_path) / "id_rsa.pub"):
```

Comment:
This should use the same Path object to avoid recreating it.

Reply:
done